### ISSUE TITLE
[Critical] Function cannot be performed by keyboard alone

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#55]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87493 |
| **Checkpoint** | 2.1 Keyboard Accessible › Keyboard Navigation (2.1.1.a) |
| **Impact** | Critical |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 01 Dashboard |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/dashboard> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 05/01/26 at 1:45 pm |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | a3c.access@gmail.com |

## Description

There is no way to perform the function using only the keyboard on the same screen or on a qualifying conforming alternate version.
Pattern. A mouse user can click on any of the Recent Work Items files, but you cannot do the same with the keyboard alone. Also occurs in Recent Notification sections. See Work Items User Experience Video at 4:50.

## Recommendation

Rule : 
Functionality MUST be available using the keyboard, unless the functionality cannot be accomplished in any known way using a keyboard.

How To Fix: 
Fix this issue by ensuring the component can be used by the keyboard. This includes ensuring controls are:
1. Focusable with a keyboard (i.e. can be reached using the Tab key or Arrow keys on a keyboard), and
2. Operable with a keyboard (e.g. links are activated with the Enter key, buttons are activated with Enter and Spacebar, checkboxes are activated with Spacebar, dragging and swiping actions have a keyboard alternative, etc.)

The best way to do this is to use native HTML elements when HTML equivalents exist (e.g. anchor, button, select, text area, and input elements). When custom (non native HTML) controls have to be used, you must ensure they can be focused with a keyboard (often using tabindex="0") and operated with a keyboard (using JavaScript event handlers). The ARIA Authoring Practices Guide provides information on expected keyboard interaction patterns for custom controls and widgets and working examples (see References below).

Reference: 
ARIA Authoring Practices Guide Patterns: https://www.w3.org/WAI/ARIA/apg/patterns/

Background: 
Some people cannot use a mouse due to vision or motor disabilities. Content that can be operated with a mouse must also be made operable with a keyboard. When content is operable through a keyboard, it becomes operable by a variety of assistive technologies such as speech input software, sip-and-puff software, on-screen keyboards, scanning software, and alternate keyboards.

## Source Code

```
<span class="is-grey-dark">Standard-bag-1.tar</span>
<span class="is-grey-dark">object_100.tar</span>
<div class="text-xs is-grey-dark is-dashboard-action">Your APTrust password has been changed</div>
```

## Screenshots

![dashboard_keyboard nav](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87493/dashboard_keyboard_nav.png)

**Comment by mjiori** [comment]
Ablr verified this fix in demo: Marked as "Fixed, RJ, 8/17/26"

The fix is now in prod. Closing.
